contracts/modules/staking/PooledStaking.sol

```diff
@@ -102,15 +108,15 @@ contract PooledStaking is MasterAware, IPooledStaking {
   uint public MIN_UNSTAKE;       // Forbid unstake of small amounts to prevent spam
   uint public UNSTAKE_LOCK_TIME; // Lock period in seconds before unstaking takes place
 
-  mapping(address => Staker) stakers;     // stakerAddress => Staker
+  mapping(address => Staker) public stakers;     // stakerAddress => Staker
 
   // temporary variables
-  uint contractStaked;   // used when processing burns and rewards
-  uint contractBurned;   // used when processing burns
-  uint contractRewarded; // used when processing rewards
+  uint public contractStaked;   // used when processing burns and rewards
+  uint public contractBurned;   // used when processing burns
+  uint public contractRewarded; // used when processing rewards
 
   // list of stakers for all contracts
-  mapping(address => address[]) contractStakers;
+  mapping(address => address[]) public contractStakers;
```

Comment:
the code on etherscan doesn't contain "public"

Reply:
I know but it's the default when not specified. The bytecode doesn't change but the linter complains about it if it's not.